### DISCUSSION TITLE
Does dragonfly support key value weed out？

### DISCUSSION THREAD

**Discussion by fansehep** [dragonflydb/dragonfly#331] [Q&A] [answered]
hello，guys，i need a cache cluster. we did some research，find memcached is fit. but it seems this project is morden and great.

Can it be used in a production environment and support kv
elimination strategy？

**Comment by odedponcz** [comment, marked as answer]
Hello @fansehep 
Dragonfly is used in production as a cache. You can read more about Dragonfly's cache design here: https://dragonflydb.io/blog/2022/06/23/cache_design/ 
I suggest that you will join our discord channel if you have more questions https://discord.gg/HsPjXGVH85
